README.md

```diff
@@ -15,9 +83,164 @@ cp example/busybox /opt/ansible/roles/
+
+```yaml
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: app-operator
+rules:
+- apiGroups:
+  - app.example.com
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  verbs:
+  - "*"
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - daemonsets
+  - replicasets
+  - statefulsets
+  verbs:
+  - "*"
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: default-account-app-operator
+subjects:
+- kind: ServiceAccount
+  name: default
+roleRef:
+  kind: Role
+  name: app-operator
+  apiGroup: rbac.authorization.k8s.io
+```
+
+##### Install the CRD into the Cluster
+The CRD or Custom Resource Definition is a key extension point in kubernetes. Here you define a new resource type, Kubernetes will handle saving and persisting this resource definition. Here is some documentation on CRDs: https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/
+An example below will work with the busybox role and rbac roles above.
+```yaml
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: databases.app.example.com
+spec:
+  group: app.example.com
+  names:
+    kind: Database
+    listKind: DatabaseList
+    plural: databases
+    singular: database
+  scope: Namespaced
+  version: v1alpha1
+```
+
+Some things to note are that CRDs have two sections, the Spec section and the Status section. The Spec is used by the user creating the CR (custom resource). This is where a user should define the parameters for the role. The Status field is used by the operator to handle the “state” of the resource. This should not be touched by the user. There are two other fields on a CR the ObjectMeta and TypeMeta which all kubernetes objects share.
+
+##### Operator Deployment
+To deploy the operator, you should create this to manage the operator pod. The below deployment with the above code should start a running operator! Backed by Ansible!
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ansible-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: ansible-operator
+  template:
+    metadata:
+      labels:
+        name: ansible-operator
+    spec:
+      containers:
+        - name: ansible-operator
+          image: docker.io/shurley/busybox-ansible-operator
```

Comment:
Is this still the authoritative image location? I saw mention of pushing an image to quay?